nservicebus/azure-service-bus/configuration/full.md

```diff
@@ -110,6 +110,7 @@ The following settings determine how NServiceBus will connect to Azure Service B
 
  * `NumberOfClientsPerEntity(int)`: NServiceBus maintains a pool of receive and send clients for each entity. This setting determines how big that pool is. Defaults to 5.
  * `ConnectivityMode(ConnectivityMode)`: Determines how NServiceBus connects to Azure Service Bus, using TCP or HTTP. Defaults to TCP.
+ * `TransportType(TransportType)`: Determines what transport protocol NServiceBus is using for Azure Service Bus, `TransportType.NetMessaging` or `TransportType.Amqp`. Defaults to `TransportType.NetMessaging`.
```

Comment:
@SimonCropp not sure, what you're refering to. But here are both answers: 
* the API as a whole is new for V7 and so is this setting
* NetMessaging has been the protocol used in the past, in theory it should have been possible for users to swithc to amqp using the connectionstring notation, but I don't think anyone ever did that. Going forward AMQP will become the defacto standard for ASB clients, so we'll have to transition eventually as well, but as long as there is feature imparity we'll stick to netmessaging as default (but support amq)